function.rb

```diff
@@ -0,0 +1,9 @@
+
+def function(n)
+  if n == 0
+    1
+  else
+    n * function(n-1)
+  end
```

Comment:
Could this be possibly a ternary? 
```suggestion
  n == 0 ? 1 : n * function(n - 1)
```